webapp/vue-portlet/src/main/webapp/suggestions-people-space/components/ExoSuggestionsPeopleAndSpace.vue

```diff
@@ -0,0 +1,147 @@
+              </transition-group>
+            </v-list>
+            <v-list v-if="spacesSuggestionsList.length > 0 && suggestionsType !== 'people'" dense class="suggestions-list space-list py-4 mx-4">
+              <transition-group name="fade" tag="div">
+                <exo-user-avatar
+                  v-for="space in spacesToDisplay"
+                  :key="space"
+                  :fullname="space.displayName"
+                  :size="iconSize"
+                  :avatarurl="space.spaceAvatarUrl"
+                  :tile="true"
+                  class="mx-auto py-2">
+                  <template slot="subTitle">
+                    {{ space.members }} {{ $t('spacemember.Label') }}
+                  </template>
+                  <template slot="actions">
+                    <v-btn-toggle class="transparent" dark>
+                      <a
+                        text
+                        icon
+                        small
+                        min-width="auto"
+                        class="px-0 suggestions-btn-action connexion-accept-btn"
+                        @click="joinSpace(space)">
+                        <i class="uiIconPlusLight"></i>
+                      </a>
+                      <a
+                        text
+                        small
+                        min-width="auto"
+                        class="px-0 suggestions-btn-action connexion-refuse-btn ml-2"
+                        @click="ignoredSuggestionSpace(space)">
+                        <i class="uiIconCloseCircled tertiary-color"></i>
+                      </a>
+                    </v-btn-toggle>
+                  </template>
+                </exo-user-avatar>
+              </transition-group>
+            </v-list>
+          </v-card>
+        </v-flex>
+      </v-layout>
+    </v-flex>
+  </v-app>
+</template>
+<script>
+import * as suggestionsServices from '../suggestionsServices.js';
+export default {
+  props: {
+    suggestionsType: {
+      type: String,
+      default: null,
+    },
+  },
+  data () {
+    return {
+      peopleSuggestionsList: [],
+      spacesSuggestionsList: [],
+      currentUser: ''
+    };
+  },
+  computed : {
+    peoplesToDisplay() {
+      return this.peopleSuggestionsList.slice(0, 2);
+    },
+    spacesToDisplay() {
+      return this.spacesSuggestionsList.slice(0, 2);
+    }
+  },
+  created() {
+    this.initPeopleSuggestionsList();
+    this.initSpaceSuggestionsList();
+  },
+  methods : {
+    initPeopleSuggestionsList() {
+      suggestionsServices.getPeopleSuggestions().then(data => {
+        this.peopleSuggestionsList = data.items;
+        this.currentUser = data.username;
+      });
+    },
+    initSpaceSuggestionsList() {
+      suggestionsServices.getSpaceSuggestions().then(data => {
+        this.spacesSuggestionsList = data.items;
+      });
+    },
+    connectionRequest(item) {
+      suggestionsServices.sendConnectionRequest(item.suggestionId);
```

Comment:
This is a `Promise` so next line should be in `then`, isn't it ?
(Same for other methods)

Reply:
Okey done!